### PR TITLE
Add local links in the intro page

### LINKED ISSUE #177: Suggestion for intro page: Add each local page after translations include:
Hello 👋,

I'd like to suggest a very small thing to improve the intro page. Under the welcome title, there is a sentence that describes the list of translations. 

```
 Some of these translations include: Việt ngữ, 简体中文, 繁體中文, Español, Slovenčina, 日本語, Polski, Português, Русский, Bahasa Indonesia, Bahasa Melayu, Українською, 한국어, Italiano, Deutsch, বাংলা, Türkçe, ภาษาไทย, and Ελληνικά.
```
As following old ver web, it would be good to add a local link to each language text to switch local pages more easily. e.g.) 

Screenshot 2021-10-28 at 15 36 22

I applied css style `<a href="/vi" class="text-purple dark:text-purple-dark underline">Việt ngữ</a>` to highlight a link.

### PR DESCRIPTION
**Pull request by sujinleeme**
fixes #177 

This PR is to add links inside the local section of the intro page.

### CODE REVIEW

File: lib/school_house_web/templates/page/index.html.heex [hunk 1]
```diff
@@ -8,9 +8,29 @@
         Elixir School is the premier destination for people looking to learn and master the Elixir programming language.
         Whether you’re a seasoned veteran or this is your first time, you’ll find what you need in lessons and auxiliary resources.
       </p>
-      <p class="leading-relaxed">
-        Through the hard work of volunteers Elixir School has been translated to many languages. Some of these translations include: Việt ngữ, 简体中文, 繁體中文, Español, Slovenčina, 日本語, Polski, Português, Русский, Bahasa Indonesia, Bahasa Melayu, Українською, 한국어, Italiano, Deutsch, বাংলা, Türkçe, ภาษาไทย, and Ελληνικά.
-        <br/>
+      <p class="mb-6 leading-relaxed prose">
+        Through the hard work of volunteers Elixir School has been translated to many languages. Some of these translations include:
```
[the remark this answers is not in the document]
Comment (sujinleeme) [reply lib/school_house_web/templates/page/index.html.heex:12]: I added dark:prose-dark too in here 3ad59e8

File: assets/tailwind.config.js [hunk 2]
```diff
@@ -148,9 +149,10 @@ module.exports = {
                         },
                         a: {
                             color: theme('colors.brand-purple-100'),
+                            fontWeight: 'inherit',
+                            textDecoration: 'none',
                             '&:hover': {
-                                'background-color': theme('colors.brand-purple-100'),
-                                color: colors.white,
+                                textDecoration: 'underline',
                             }
```
[the remark this answers is not in the document]
Comment (sujinleeme) [reply assets/tailwind.config.js]: Yeah, I got it now. I addressed the comment in here 6ab4b63.

File: lib/school_house_web/templates/page/index.html.heex [hunk 3]
```diff
@@ -8,9 +8,29 @@
...
+        <a href="/id" title="Indonesian">Bahasa Indonesia</a>,
+        <a href="/ms" title="Malay">Bahasa Melayu</a>,
+        <a href="/uk" title="Ukrainian">Українською</a>,
+        <a href="/ko" title="Korean">한국어</a>,
+        <a href="/it" title="Italian">Italiano</a>,
+        <a href="/de" title="German">Deutsch</a>,
+        <a href="/bn" title="Bengali">বাংলা</a>,
+        <a href="/tr" title="Turkish">Türkçe</a>,
+        <a href="/th" title="Thai">ภาษาไทย</a>, and
+        <a href="/el" title="Modern Greek">Ελληνικά</a>.
```
Comment (kinson) [inline lib/school_house_web/templates/page/index.html.heex:31]: I have never used the `title` attribute - that's handy!

**Review by kinson — APPROVED** [review]
Looks great! Thanks for the idea and the execution 💪🏼

**Pull request merged**